kpi/serializers/v2/asset_counts.py

```diff
@@ -14,12 +15,9 @@ class Meta:
         fields = ('daily_submission_counts', 'total_submission_count')
 
     def get_daily_submission_counts(self, asset):
-        today = timezone.now().date()
+        today = timezone.now().date() + datetime.timedelta(days=1)
         request = self.context['request']
-        if 'days' in self.context:
-            start_date = today - relativedelta(days=int(self.context['days']))
-        else:
-            start_date = today - relativedelta(days=31)
+        start_date = today - datetime.timedelta(days=int(self.context['days']))
```

Comment:
We should validate `self.context['days']` is an integer and raise a `ValidationError` if it's not the case.

Reply:
The number of days gets turned into a string when passed as a URL parameter, so it needs to be converted regardless, would it be better to convert it to an integer in the viewset?

Reply:
@JacquelineMorrissette , I meant we should validate the value of `self.context['days']` is digit, not characters before cast to `int()`. If it's not then raise the ValidationError. 
So no, do not convert it in the viewset. Let the serializer do the job is the validation passes.